docs/pages/database-access/guides/postgres-aws.mdx

```diff
@@ -76,52 +76,12 @@ GRANT rds_iam TO alice;
 See [Creating a database account using IAM authentication](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/UsingWithRDS.IAMDBAuth.DBAccounts.html)
 for more information.
 
-## Configure Teleport
+## Setup Teleport Auth and Proxy Services
 
-Teleport Database Access is available starting from `6.0` release.
+Teleport Database Access for PostgreSQL on AWS is available starting from `6.0`
```

Comment:
Might rephrase this to: `Teleport Database Access for PostgreSQL on AWS is available 6.0+...` 

Previously we had one or two comments about versioning that seemed to indicate that the functionality was available in only one version (and not subsequent ones) or that needed to better indicate this for other reasons.

Reply:
I think that "starting from" indicates clearly enough that it's available in all subsequent releases as well but let me know if you think otherwise.